src/condor_utils/AWSv4-utils.h

```diff
@@ -28,33 +24,4 @@ generate_presigned_url( const classad::ClassAd & jobAd,
 	CondorError & err );
 }
 
-// Functions used internally by generate_presigned_url() and the Amazon GAHP.
-
-namespace AWSv4Impl {
-
-std::string
-pathEncode( const std::string & original );
-
-std::string
-amazonURLEncode( const std::string & input );
-
-std::string
-canonicalizeQueryString( const std::map< std::string, std::string > & qp );
-
-void
-convertMessageDigestToLowercaseHex( const unsigned char * messageDigest,
```

Comment:
This function (renamed) probably deserves a spot in stl_string_utils.*, but not necessary for this PR.